logrotate/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ SET(DIR_3RDPARTY ${LogRotate_SOURCE_DIR}/../3rdparty)
 
 MESSAGE(" PROJECT_SRC = : ${PROJECT_SRC}")
 INCLUDE_DIRECTORIES(${PROJECT_SRC})
+INCLUDE_DIRECTORIES(${PROJECT_SRC}/../../common)
```

Comment:
This should not be needed.  If it is then something wonky is happening with CMake. It should be fixed with Cmake instead.